src/components/JobCard/JobCard.tsx

```diff
@@ -0,0 +1,107 @@
+import { Grid, Avatar, Typography, Button, Chip, Card, CardContent, Stack } from '@mui/material'
+import BusinessIcon from '@mui/icons-material/Business'
+import LocationOnIcon from '@mui/icons-material/LocationOn'
+import AccountBalanceWalletIcon from '@mui/icons-material/AccountBalanceWallet'
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+
+
+interface JobCardProps {
+    title: string;
+    company: string;
+    logoUrl: string;
+    jobPostingUrl: string;
+    location:string,
+    salary: string;
+    publicationDate:string;
+    tags: string[];
+}
+
+const JobCard: FC<JobCardProps> = ({ title, company, logoUrl, jobPostingUrl, location, salary, publicationDate, tags }) => {
+    const handleReadMoreClick = () => {
+        if (jobPostingUrl) {
+            window.open(jobPostingUrl, '_blank', 'noopener,noreferrer')
+        } else {
+            console.error('No URL provided')
+        }
+    }
+
+    return (
+        <Stack spacing={1} alignItems="center">
+            <Card sx={{ bgcolor: '#512DA8', color: 'white', width: '100%' }}>
+                <CardContent>
+                    <Grid container spacing={1} alignItems="center">
+                        <Grid item xs={12} sm={12} md={2} display="flex" justifyContent="center">
+                            <Avatar
+                                sx={{ bgcolor: 'grey', width: 80, height: 80, borderRadius: 2 }}
+                                alt={`${company} logo`}
+                                src={logoUrl}
+                            />
+                        </Grid>
+                        <Grid item xs={12} sm={12} md={8}>
+                            <Typography variant="h6" component="div" sx={{ textAlign: 'left' }}>
+                                {title}
+                            </Typography>
+                            <Grid container spacing={2}>
+                                <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: { xs: 'flex-start', sm: 'center' }, alignItems: 'center' }}>
+                                    <BusinessIcon fontSize="small" />
+                                    <Typography fontSize="medium" sx={{ ml: 1 }}>{company}</Typography>
+                                </Grid>
+                                <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: { xs: 'flex-start', sm: 'center' }, alignItems: 'center' }}>
+                                    <LocationOnIcon fontSize="small" />
+                                    <Typography fontSize="medium" sx={{ ml: 1 }}>{location}</Typography>
+                                </Grid>
+                                <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: { xs: 'flex-start', sm: 'center' }, alignItems: 'center' }}>
+                                    <AccountBalanceWalletIcon fontSize="small" />
+                                    <Typography fontSize="medium" sx={{ ml: 1 }}>{salary}</Typography>
```

Comment:
For consistency across our site, it's best to stick to the types defined in our theme:

http://localhost:5173/#/theme

```suggestion
                                    <Typography variant='body2' sx={{ ml: 1 }}>{salary}</Typography>
```